docs/data-sources/cce_cluster_kubeconfig_v3.md

```diff
@@ -0,0 +1,35 @@
+---
+subcategory: "Cloud Container Engine (CCE)"
+---
+
+# opentelekomcloud_cce_cluster_kubeconfig_v3
+
+Use this data source to get details about a cluster kubeconfig file from OpenTelekomCloud.
```

Comment:
It should be not `details about a cluster kubeconfig file`, but `a cluster kubeconfig file`, or something like that.